ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/Workbenches.resource

```diff
@@ -422,19 +423,21 @@ Check Launched Workbench Is The Correct One
 Get Openshift Notebook CR From Workbench
     [Documentation]    Retrieves name of Notebook CR corresponding to a workbench
     [Arguments]     ${workbench_title}      ${namespace}
-    ${rc}  ${cr_name}=    Run And Return Rc And Output   oc get notebook -n ${namespace} -o jsonpath='{.items[?(@.metadata.annotations.openshift\\.io/display-name=="${workbench_title}")].metadata.name}'
-    RETURN    ${rc}    ${cr_name}
+    ${result}=    Run Process
+    ...    oc get notebook -n ${namespace} -o jsonpath\='{.items[?(@.metadata.annotations.openshift\\.io/display-name\=\="${workbench_title}")].metadata.name}'
+    ...    shell=yes
+    RETURN    ${result.rc}    ${result.stdout}
 
 Start Workbench Should Fail
     [Documentation]    Checks the workbench fails to start
-    [Arguments]    ${workbench_title}
+    [Arguments]    ${prj_title}    ${workbench_title}
+    Wait Until Project Is Open    project_title=${prj_title}
```

Comment:
why? this should be already open when this kw is supposed to be used

Reply:
>It caused instability in the (only single) test that used it


meaning that the project wasn't open? 

> verifying that project within the KW


it's redudant, in order to run this kw you already ran the kw to open the project which already waits for the project to be opened.

Reply:
> meaning that the project wasn't open?

Yes.
But I can move the logic outside the KW, which means that we will need to verify the project of the workbench in each test, before using that KW.

Reply:
I restored to the original logic, and removed `Wait Until Project Is Open`